File: java/Problem1134.java
import java.util.Scanner;

public class Problem1134 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int alcool = 0;
        int gasolina = 0;
        int diesel = 0;

        while (true) {
            int item = sc.nextInt();
            if (item > 0 && item < 5) {
                if (item == 1) {
                    alcool++;
                }
                if (item == 2) {
                    gasolina++;
                }
                if (item == 3) {
                    diesel++;
                }
                if (item == 4) {
                    break;
                }
            }
        }

        System.out.println("MUITO OBRIGADO");
        System.out.printf("Alcool: %d\n", alcool);
        System.out.printf("Gasolina: %d\n", gasolina);
        System.out.printf("Diesel: %d\n", diesel);

        sc.close();
    }
}

File: python/problem1134.py
alc = 0
gas = 0
die = 0
while True:
    x = int(input())
    
    if x == 1:
        alc += 1
    elif x == 2:
        gas += 1
    elif x == 3:
        die += 1
    elif x == 4:
        break

print('MUITO OBRIGADO')
print('Alcool: %d' % alc)
print('Gasolina: %d' % gas)
print('Diesel: %d' % die)
